src/rules/prefer-locator.test.ts

```diff
@@ -6,28 +6,118 @@ runRuleTester('prefer-locator', rule, {
     {
       code: `
           async function test() {
-              await page.fill();
+              await page.fill('input[type="password"]', 'password');
           }
       `,
       errors: [
         {
-          column: 15,
-          endColumn: 32,
+          column: 21,
+          endColumn: 68,
           endLine: 3,
           line: 3,
-          messageId: 'avoidAwaitPageMethods',
+          messageId: 'preferLocator',
+        },
+      ],
+      output: null,
+    },
+    {
+      code: `
+          async function test() {
+              await page.dblclick('xpath=//button');
+          }
```

Comment:
There is a `test` util to wrap code in test files in a playwright test block

Reply:
I've moved all the tests to use the test method, thanks. BTW: I'm on holiday from the 27th of Sept and won't have access to a PC for 5 weeks, so I hope to wrap this up before I head away.